simexpal/launch/common.py

```diff
@@ -374,7 +374,12 @@ def close(self):
 
 		elapsed = time.perf_counter() - start
 		if manifest.timeout is not None and elapsed > manifest.timeout:
-			child.send_signal(signal.SIGXCPU)
+			# programs can catch SIGXCPU and keep running
+			# kill command ensures that the time limit is respected (with a grace period)
```

Comment:
Coding style nitpick: comments should be proper sentences (capitalized and end with a dot).